<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actorEdit</class>
 <widget class="QDialog" name="actorEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lastnameLabel">
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="yearLabel">
     <property name="text">
      <string>Год</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5" colspan="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="monthLabel">
     <property name="text">
      <string>Месяц</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="birthdayLabel">
     <property name="text">
      <string>Дата рождения</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Сбросить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="dayLabel">
     <property name="text">
      <string>День</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="filmsLabel">
     <property name="text">
      <string>ID фильмов через ;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="5">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QLineEdit" name="lastnameEdit"/>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QLineEdit" name="filmsEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="dayEdit"/>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="monthEdit"/>
   </item>
   <item row="3" column="6">
    <widget class="QLineEdit" name="yearEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
